Gemfile

```diff
@@ -1,26 +1,29 @@
-# frozen_string_literal: true
```

Comment:
The `.Rhistory` file about shouldn't be committed and should instead be ignored as part of your global gitignores